src/ratchet.rs

```diff
@@ -35,22 +35,22 @@ impl Nixpkgs {
 /// The ratchet value for a top-level package
 pub struct Package {
     /// The ratchet value for the check for non-auto-called empty arguments
-    pub manual_definition: RatchetState<ManualDefinition>,
+    pub manual_definition: State<ManualDefinition>,
```

Comment:
If commited to this change, make it impossible for it to be used without the word ratchet. Last thing we want is for someone to be reading this and have to put in a cognitive effort to determine which specific state is being referred to.